Regression/WarpX-tests.ini

```
@@ -209,6 +209,22 @@ compileTest = 0
 doVis = 0
 analysisRoutine = Examples/Tests/btd_rz/analysis_BTD_laser_antenna.py
 
+[collider_diagnostics]
+buildDir = .
+inputFile = Examples/Tests/collider_relevant_diags/inputs_3d_multiple_particles
+runtime_params = warpx.abort_on_warning_threshold=high
```

```suggestion
runtime_params = warpx.abort_on_warning_threshold=medium
```
If the test is expected to generate only `low` priority warnings I would suggest setting this threshold to `medium`

---

Here we have to keep `high` because this other warning, unrelated to this PR, gets triggered:
```
[medium][QED] The built-in Quantum Synchrotron table will be used.
              This low resolution table is intended for testing purposes only.
```